src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -25,6 +25,33 @@ extern "C" {
 #pragma clang diagnostic pop
 #endif
 
+/* TODO: Potentially move these call to PIC instead. Moving to PIC in the future would not cause any backward compatibility issues */
+#define PROFILE_CALL(f, msg)                                                                                                                         \
+    do {                                                                                                                                             \
+        UINT64 startTime = GETTIME();                                                                                                                \
```

Comment:
Variable declaration in a macro like this means the variable can't be declared at top of function, which is troublesome for some C compilers.

Our other macros like this one that rely on `retStatus` come with the requirement that the variable be declared outside the scope of this macro.

Reply:
We could also rename it to show that it's a variable for this macro.